mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Req.MixProject do
   use Mix.Project
 
-  @version "0.3.1"
+  @version "0.3.2"
```

Comment:
Let's not change versions (or changelogs etc) on PRs as we don't know when we'll release it, maybe it will be 0.3 or maybe 0.4.

```suggestion
  @version "0.3.1"
```